### PR TITLE
Product cards

### LINKED ISSUE #7: Product cards
https://www.bitdefender.com.au/solutions/

Identify what cards exists on all pages https://adobe-dx-support.slack.com/archives/C05E0MH7FRV/p1688718296627479?thread_ts=1688123219.642389&cid=C05E0MH7FRV and document the variations.

Then start working to implement the different card variants with focus on CSS/JS. We don't know yet where the content will come from. Assuming from external API.

Example of documentation: https://github.com/hlxsites/mammotome/blob/main/docs/blocks/cards.md

**Card variations** 

Red-marked values come from the web service, the rest is content.

Screenshot 2023-07-19 at 07 24 23
Screenshot 2023-07-19 at 07 25 26
Screenshot 2023-07-19 at 07 21 47

**Tasks**

- [x] Implement functionality for fetching data
- [x] Implement a block for rendering cards
- [ ] Propose new content structure to a customer

### PR DESCRIPTION
**Pull request by vtsaplin**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #7

Test URLs:
- Before: https://main--bitdefender--hlxsites.hlx.page/solutions/
- After: https://product-cards--bitdefender--hlxsites.hlx.page/solutions/

### CODE REVIEW

File: blocks/products/products.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+const FETCH_URL = 'https://www.bitdefender.com/site/Store/ajax';
+
+/**
+ * Fetches a product from the Bitdefender store.
+ * @param code The product code
+ * @param variant The product variant
+ * @returns {Promise<*>}
+ */
+async function fetchProduct(code = 'av', variant = '1u-1y') {
```
Comment (rhudea) [inline blocks/products/products.js:11]: Should this be outside of products since it might be used from other blocks (like product comparison table) ?

**Comment by rhudea** [comment]
The buttons content is not quite right:
image

**Review by rhudea — APPROVED** [review]
small nitpicks
but otherwise LGTM

File: scripts/fetch-product.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const FETCH_URL = 'https://www.bitdefender.com/site/Store/ajax';
+
+/**
+ * Fetches a product from the Bitdefender store.
+ * @param code The product code
+ * @param variant The product variant
+ * @returns {Promise<*>}
+ */
+async function fetchProduct(code = 'av', variant = '1u-1y') {
```
[the remark this answers is not in the document]
Comment (nicolasbarbe) [reply scripts/fetch-product.js:9]: I merged everything in utils to reduce the number of files.

**Pull request merged**